pkg/resources/testdata/TestAcc_TagAssociation/issue1202/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "snowflake_tag" "test" {
+  name     = var.tag_name
+  database = var.database
+  schema   = var.schema
+}
+
+resource "snowflake_table" "test" {
+  name     = var.table_name
+  database = var.database
+  schema   = var.schema
+  column {
+    name = "test_column"
+    type = "VARIANT"
+  }
+}
+
+resource "snowflake_tag_association" "test" {
+  object_name = "\"${var.database}\".\"${var.schema}\".\"${var.table_name}\""
+  object_type = "TABLE"
+  tag_id      = snowflake_tag.test.id
+  tag_value   = "v1"
```

Comment:
`depends_on` is missing (we require `snowflake_table` to be created first)

Reply:
name     = snowflake_table.test.name
this is already an implicit dependency. actually you do not need to do depends_on

Reply:
yep, but it weren't there, so any dependency was indeed needed